spec/acceptance/nodesets/centos_with_hirs.yml

```diff
@@ -0,0 +1,117 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
+    hypervisor = ENV['BEAKER_HYPERVISOR']
+  else
+    hypervisor = 'vagrant'
+  end
+-%>
+HOSTS:
+  el7-boltserver:
+    roles:
+      - default
+      - boltserver
+      - target
+      - hirs
+      - tpm_1_2
+    platform:   el-7-x86_64
+    box:        centos/7
+    hypervisor: <%= hypervisor %>
+    yum_repos:
+      epel:
+        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch'
+        gpgkeys:
+          - https://getfedora.org/static/352C64E5.txt
```

Comment:
I think this needs to be changed to https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7

Reply:
Changed to https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-$releasever in all nodesets.